core/services/log/registrations.go

```diff
@@ -50,6 +50,8 @@ type (
 	// metadata structure maintained per listener, used to avoid double-sends of logs
 	listenerMetadata struct {
 		opts                     ListenerOpts
+		filters                  [][]Topic
+		noFilter                 bool
```

Comment:
Is it not sufficient to have an empty filter list mean no filters?

Reply:
oh yeah very good point